proto/primitive/session_context.proto

```diff
@@ -25,6 +30,6 @@ message SessionContext {
 	string tenancy_name = 3;
 	string user_name = 4;
 	.holms.types.iam.StaffMemberIndicator user = 5;
-	repeated Capability capabilities = 6;
+	repeated UserEntitlements user_entitlements = 6;
```

Comment:
The list of entitlements already appears in the token, so this technically duplicates that.

No problem, and in fact, I would even prefer if we opt to keep the token totally client-opaque, as it makes it easier to change later. I think this is the approach you've taken so far, let's keep it that way.

OTOH you should decide whether you'll use the JSON-encoded version of the entitlements, which you have to develop anyway because you need to represent them on the token, here, or you'd rather keep that an implementation detail that doesn't leak out. If you're really serious about keeping it as an implementation detail, can you move the entire JWToken and all that business into application?

I'm actually not clear on what all uses jwtoken/why it's in this repo, if tokens are only handled/processed (apart from being opaque strings) server-side? Are they interpreted in the client somehow?